packages/server/src/main/java/software/uncharted/terarium/hmiserver/utils/rebac/ReBACService.java

```diff
@@ -491,11 +491,15 @@ public void removeRelationship(
 		final SchemaObject who,
 		final SchemaObject what,
 		final Schema.Relationship relationship
-	) throws Exception, RelationshipAlreadyExistsException {
+	) throws Exception {
 		userCache.invalidate(who.id);
 		invalidatePermissionCache(who, what);
 		final ReBACFunctions rebac = new ReBACFunctions(channel, spiceDbBearerToken);
-		CURRENT_ZED_TOKEN = rebac.removeRelationship(who, relationship, what);
+		try {
+			CURRENT_ZED_TOKEN = rebac.removeRelationship(who, relationship, what);
+		} catch (RelationshipAlreadyExistsException ignore) {
```

Comment:
Seriously - RelationshipAlreadyExistsException is the exception for when a relationship does not exist when removing - lol - oh well silliness

Reply:
I mean, that's why I ignored it, I want to remove it but it doesn't exist, good for me regardless. I am not going to check if it exist before deleting it?

Reply:
Exactly - I was laughing at the curious name of the Exception (I created) but it is because the underlying exception is about a failure to create a relationship, even though the request is to delete the relationship